rca/project_styleguide/templates/patterns/pages/landingpage/landing_page--research.html

```diff
@@ -103,13 +103,16 @@ <h2 class="section__heading section__heading--primary heading heading--four">
                     <div class="section__row section__row--first section__row--last">
                         {% include "patterns/organisms/image-video-block/image-video-block.html" with modal=false curriculum=false subheading=false has_meta=True heading=featured_image.title meta_heading=featured_image.subtitle meta_copy=featured_image.description href=featured_image.get_link_url image=featured_image.image modifier="tight-heading" %}
                     </div>
+
+                    {% if slideshow_block or page.cta_block or page.news %}
```

Comment:
These and the other if statements for the notches are the fix for the issue where a black/white bar is left behind.

> 2) not using "text promo" and "related content" components leaves odd white bar on page (see screenshot)